src/WordSelector/WordSelector.js

```diff
@@ -28,11 +28,12 @@ export class WordSelector extends Component {
           <h2>Type a word for your opponent to guess!</h2>
           <form>
             <label>
-              <input type="text" name="name" onChange={this.updateChange} />
+              <input type="text" name="name" onChange={this.updateChange} placeholder="Name"/>
```

Comment:
Nice use of placeholder here! I was thinking about it, the onClick in 37 below might need to get changed onSubmit (I think that's what it's called). That way users can hit "enter"/"return" to submit this as well.